doc/2/api/controllers/server/capabilities/index.md

```diff
@@ -0,0 +1,74 @@
+---
+code: true
+type: page
+title: capabilities
+---
+
+# capabilities
+
+
+
+Returns the current Kuzzle capabilities.
+
+This action is accessible by all users, the purpose of this action is to inform the client what kind of limits or behavior they should expect from Kuzzle.
+
```

Comment:
You should add the list of which properties are returned and what information they are giving